src/components/cart/CartContents/CartContents.tsx

```diff
@@ -0,0 +1,98 @@
+import { useNavigate } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+import { styled } from 'styled-components';
+import { Suspense, useState } from 'react';
+import CartList from '../CartList/CartList';
+import PaymentAmount from '../PaymentAmount/PaymentAmount';
+import getCartLength from '../../../globalState/selectors/getCartLength';
+import CouponInfo from '../../../types/coupon';
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export